Source/CesiumRuntime/Private/CesiumCreditSystem.cpp

```diff
@@ -41,20 +42,25 @@ ACesiumCreditSystem* findValidDefaultCreditSystem(ULevel* Level) {
         TEXT("No valid level for findValidDefaultCreditSystem"));
     return nullptr;
   }
+#if ENGINE_VERSION_5_4_OR_HIGHER
+  TArray<TObjectPtr<AActor>>& Actors = Level->Actors;
+  TObjectPtr<AActor>* DefaultCreditSystemPtr;
```

Comment:
Might be good form to set it explicitly to nullptr here?

```suggestion
  TObjectPtr<AActor>* DefaultCreditSystemPtr = nullptr;
```

Reply:
Yeah, avoiding having a pointer in an uninitialized state sounds cleaner.
Checked in some changes that avoids this completely (put an alias declaration in the #if instead)